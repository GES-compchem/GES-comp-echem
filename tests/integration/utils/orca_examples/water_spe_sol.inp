%pal
  nprocs 4
end

%maxcore 350

! PBE def2-SVP 
! RIJCOSX def2/J

%CPCM
  SMD True
  SMDsolvent "water"
end

* xyzfile 0 1 water.xyz
