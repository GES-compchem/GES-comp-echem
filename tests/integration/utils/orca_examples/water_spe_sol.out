
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ######   ,####,        
            ##       ##  ##   ,#'  ##            #'  '#       #        #'  '#        
            ##       ##  #######   ##           ,######,      #####,   #    #        
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,         ##   #,  ,#        
             '#######'   ##     ##  '#######'  #'      '#     #####' # '####'        



                  #######################################################
                  #                        -***-                        #
                  #          Department of theory and spectroscopy      #
                  #    Directorship and core code : Frank Neese         #
                  #        Max Planck Institute fuer Kohlenforschung    #
                  #                Kaiser Wilhelm Platz 1               #
                  #                 D-45470 Muelheim/Ruhr               #
                  #                      Germany                        #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 5.0.3 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Suceptibility
   Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
   Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum
   Ute Becker             : Parallelization
   Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Miquel Garcia          : C-PCM and meta-GGA Hessian, CC/C-PCM, Gaussian charge scheme
   Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Benjamin Helmich-Paris : MC-RPA, TRAH-SCF, COSX integrals
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Marcus Kettner         : VPT2
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : Initial AUTO-CI
   Lucas Lang             : DCDCAS
   Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC
   Dagmar Lenk            : GEPOL surface, SMD
   Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization
   Dimitrios Manganas     : Further ROCIS development; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Original ROCIS implementation
   Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density
   Barbara Sandhoefer     : DKH picture change effects
   Avijit Sen             : IP-ROCIS
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Bernardo de Souza      : ESD, SOC TD-DFT
   Georgi Stoychev        : AutoAux, RI-MP2 NMR, DLPNO-MP2 response
   Willem Van den Heuvel  : Paramagnetic NMR
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse, P. Pracht,  : VdW corrections, initial TS optimization,
                  C. Bannwarth, S. Ehlert          DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model
   Lars Goerigk                                  : TD-DFT with DH, B97 family of functionals
   V. Asgeirsson, H. Jonsson                     : NEB implementation
   FAccTs GmbH                                   : IRC, NEB, NEB-TS, DLPNO-Multilevel, CI-OPT
                                                   MM, QMMM, 2- and 3-layer-ONIOM, Crystal-QMMM,
                                                   LR-CPCM, SF, NACMEs, symmetry and pop. for TD-DFT,
                                                   nearIR, NL-DFT gradient (VV10), updates on ESD,
                                                   ML-optimized integration grids
   S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library
   Liviu Ungur et al                             : ANISO software


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 5.1.0
 For citations please refer to: https://tddft.org/programs/libxc/

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices
   BLAS/LAPACK       :  OpenBLAS 0.3.15  USE64BITINT DYNAMIC_ARCH NO_AFFINITY Zen SINGLE_THREADED
        Core in use  :  Zen
   Copyright (c) 2011-2014, The OpenBLAS Project




***************************************
The coordinates will be read from file: water.xyz
***************************************


Your calculation utilizes the SMD solvation module
Please cite in your paper:
A.V. Marenich, C.J. Cramer, D.G. Truhlar J. Phys. Chem. B, 113, (2009), 6378 

================================================================================

----- Orbital basis set information -----
Your calculation utilizes the basis: def2-SVP
   F. Weigend and R. Ahlrichs, Phys. Chem. Chem. Phys. 7, 3297 (2005).

----- AuxJ basis set information -----
Your calculation utilizes the auxiliary basis: def2/J
   F. Weigend, Phys. Chem. Chem. Phys. 8, 1057 (2006).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================



WARNING: Found flag for RIJCOSX approximation but no HF exchange needed!
  ===> : RIJCOSX  will be disabled and Split-RI-J is turned on

INFO   : the flag for use of the SHARK integral package has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = water_spe_sol.inp
|  1> %pal
|  2>   nprocs 4
|  3> end
|  4> 
|  5> %maxcore 350
|  6> 
|  7> ! PBE def2-SVP 
|  8> ! RIJCOSX def2/J
|  9> 
| 10> %CPCM
| 11>   SMD True
| 12>   SMDsolvent "water"
| 13> end
| 14> 
| 15> * xyzfile 0 1 water.xyz
| 16> 
| 17>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  O     -3.210350   -0.585040   -0.013950
  H     -2.242470   -0.618270    0.018480
  H     -3.489200   -1.249110    0.634290

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 O     8.0000    0    15.999   -6.066682   -1.105565   -0.026362
   1 H     1.0000    0     1.008   -4.237654   -1.168361    0.034922
   2 H     1.0000    0     1.008   -6.593632   -2.360476    1.198634

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     0.968993102246     0.00000000     0.00000000
 H      1   2   0     0.969000198658   103.97800340     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     1.831131588903     0.00000000     0.00000000
 H      1   2   0     1.831144999180   103.97800340     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 2 groups of distinct atoms

 Group   1 Type O   : 7s4p1d contracted to 3s2p1d pattern {511/31/1}
 Group   2 Type H   : 4s1p contracted to 2s1p pattern {31/1}

Atom   0O    basis set group =>   1
Atom   1H    basis set group =>   2
Atom   2H    basis set group =>   2
---------------------------------
AUXILIARY/J BASIS SET INFORMATION
---------------------------------
There are 2 groups of distinct atoms

 Group   1 Type O   : 12s5p4d2f1g contracted to 6s4p3d1f1g pattern {711111/2111/211/2/1}
 Group   2 Type H   : 5s2p1d contracted to 3s1p1d pattern {311/2/1}

Atom   0O    basis set group =>   1
Atom   1H    basis set group =>   2
Atom   2H    basis set group =>   2


           ************************************************************
           *        Program running with 4 parallel MPI-processes     *
           *              working on a common directory               *
           ************************************************************
------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
                           -- RI-GTO INTEGRALS CHOSEN --
------------------------------------------------------------------------------
------------------------------------------------------------------------------
                   ___                                                        
                  /   \      - P O W E R E D   B Y -                         
                 /     \                                                     
                 |  |  |   _    _      __       _____    __    __             
                 |  |  |  | |  | |    /  \     |  _  \  |  |  /  |          
                  \  \/   | |  | |   /    \    | | | |  |  | /  /          
                 / \  \   | |__| |  /  /\  \   | |_| |  |  |/  /          
                |  |  |   |  __  | /  /__\  \  |    /   |      \           
                |  |  |   | |  | | |   __   |  |    \   |  |\   \          
                \     /   | |  | | |  |  |  |  | |\  \  |  | \   \       
                 \___/    |_|  |_| |__|  |__|  |_| \__\ |__|  \__/        
                                                                              
                      - O R C A' S   B I G   F R I E N D -                    
                                      &                                       
                       - I N T E G R A L  F E E D E R -                       
                                                                              
 v1 FN, 2020, v2 2021                                                         
------------------------------------------------------------------------------


Reading SHARK input file water_spe_sol.SHARKINP.tmp ... ok
----------------------
SHARK INTEGRAL PACKAGE
----------------------

Number of atoms                             ...      3
Number of basis functions                   ...     24
Number of shells                            ...     12
Maximum angular momentum                    ...      2
Integral batch strategy                     ... SHARK/LIBINT Hybrid
RI-J (if used) integral strategy            ... SPLIT-RIJ (Revised 2003 algorithm where possible)
Printlevel                                  ...      1
Contraction scheme used                     ... SEGMENTED contraction
Coulomb Range Separation                    ... NOT USED
Exchange Range Separation                   ... NOT USED
Finite Nucleus Model                        ... NOT USED
Auxiliary Coulomb fitting basis             ... AVAILABLE
   # of basis functions in Aux-J            ...     71
   # of shells in Aux-J                     ...     25
   Maximum angular momentum in Aux-J        ...      4
Auxiliary J/K fitting basis                 ... NOT available
Auxiliary Correlation fitting basis         ... NOT available
Auxiliary 'external' fitting basis          ... NOT available
Integral threshold                          ...     1.000000e-10
Primitive cut-off                           ...     1.000000e-11
Primitive pair pre-selection threshold      ...     1.000000e-11

Calculating pre-screening integrals         ... done (  0.0 sec) Dimension = 12
Organizing shell pair data                  ... done (  0.0 sec)
Shell pair information
Total number of shell pairs                 ...        78
Shell pairs after pre-screening             ...        78
Total number of primitive shell pairs       ...       272
Primitive shell pairs kept                  ...       265
          la=0 lb=0:     28 shell pairs
          la=1 lb=0:     28 shell pairs
          la=1 lb=1:     10 shell pairs
          la=2 lb=0:      7 shell pairs
          la=2 lb=1:      4 shell pairs
          la=2 lb=2:      1 shell pairs

Calculating one electron integrals          ... done (  0.0 sec)
Calculating RI/J V-Matrix + Cholesky decomp.... done (  0.0 sec)
Calculating Nuclear repulsion               ... done (  0.0 sec) ENN=      9.084297099237 Eh

SHARK setup successfully completed in   0.1 seconds

Maximum memory used throughout the entire GTOINT-calculation: 8.7 MB


           ************************************************************
           *        Program running with 4 parallel MPI-processes     *
           *              working on a common directory               *
           ************************************************************
-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Density Functional     Method          .... DFT(GTOs)
 Exchange Functional    Exchange        .... PBE
   PBE kappa parameter   XKappa         ....  0.804000
   PBE mue parameter    XMuePBE         ....  0.219520
 Correlation Functional Correlation     .... PBE
   PBE beta parameter  CBetaPBE         ....  0.066725
 LDA part of GGA corr.  LDAOpt          .... PW91-LDA
 Gradients option       PostSCFGGA      .... off
   Density functional embedding theory  .... OFF
   NL short-range parameter             ....  6.400000
 RI-approximation to the Coulomb term is turned on
   Number of AuxJ basis functions       .... 71


General Settings:
 Integral files         IntName         .... water_spe_sol
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   10
 Basis Dimension        Dim             ....   24
 Nuclear Repulsion      ENuc            ....      9.0842970992 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Trust-Rad. Augm. Hess. CNVTRAH         .... auto
   Auto Start mean grad. ratio tolernc. ....  1.125000
   Auto Start start iteration           ....    20
   Auto Start num. interpolation iter.  ....    10
   Max. Number of Micro iterations      ....    16
   Max. Number of Macro iterations      .... Maxiter - #DIIS iter
   Number of Davidson start vectors     ....     2
   Converg. threshold I  (grad. norm)   ....   5.000e-05
   Converg. threshold II (energy diff.) ....   1.000e-06
   Grad. Scal. Fac. for Micro threshold ....   0.100
   Minimum threshold for Micro iter.    ....   0.010
   NR start threshold (gradient norm)   ....   0.001
   Initial trust radius                 ....   0.400
   Minimum AH scaling param. (alpha)    ....   1.000
   Maximum AH scaling param. (alpha)    .... 1000.000
   Orbital update algorithm             .... Taylor
   White noise on init. David. guess    .... on
   Maximum white noise                  ....   0.010
   Quad. conv. algorithm                .... NR
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... SHARK and LIBINT hybrid scheme
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 3.830e-02
Time for diagonalization                   ...    0.000 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.004 sec
Total time needed                          ...    0.009 sec

Time for model grid setup =    0.012 sec

------------------------------
INITIAL GUESS: MODEL POTENTIAL
------------------------------
Loading Hartree-Fock densities                     ... done
Calculating cut-offs                               ... done
Initializing the effective Hamiltonian             ... done
Setting up the integral package (SHARK)            ... done
Starting the Coulomb interaction                   ... done (   0.0 sec)
Reading the grid                                   ... done
Mapping shells                                     ... done
Starting the XC term evaluation                    ... done (   0.0 sec)
  promolecular density results
     # of electrons  =      9.998906425
     EX              =     -8.726226855
     EC              =     -0.315781802
     EX+EC           =     -9.042008657
Transforming the Hamiltonian                       ... done (   0.0 sec)
Diagonalizing the Hamiltonian                      ... done (   0.0 sec)
Back transforming the eigenvectors                 ... done (   0.0 sec)
Now organizing SCF variables                       ... done
                      ------------------
                      INITIAL GUESS DONE (   0.1 sec)
                      ------------------
-------------------
DFT GRID GENERATION
-------------------

General Integration Accuracy     IntAcc      ... 4.388
Radial Grid Type                 RadialGrid  ... OptM3 with GC (2021)
Angular Grid (max. ang.)         AngularGrid ... 4 (Lebedev-302)
Angular grid pruning method      GridPruning ... 4 (adaptive)
Weight generation scheme         WeightScheme... Becke
Basis function cutoff            BFCut       ... 1.0000e-10
Integration weight cutoff        WCut        ... 1.0000e-14
Angular grids for H and He will be reduced by one unit
Partially contracted basis set               ... off
Rotationally invariant grid construction     ... off

Total number of grid points                  ...    12750
Total number of batches                      ...      201
Average number of points per batch           ...       63
Average number of grid points per atom       ...     4250
Time for grid setup =    0.044 sec


--------------------
CPCM SOLVATION MODEL
--------------------
CPCM parameters:
  Epsilon                                         ...      78.3550
  Refrac                                          ...       1.3328
  Rsolv                                           ...       1.3000
  Surface type                                    ... GAUSSIAN VDW
  Epsilon function type                           ...         CPCM
Solvent:                                     ... WATER
SMD-CDS solvent descriptors:
  Soln                                       ...    1.3328
  Soln25                                     ...    1.3323
  Sola                                       ...    0.0000
  Solb                                       ...    0.0000
  Solg                                       ...    0.0000
  Solc                                       ...    0.0000
  Solh                                       ...    0.0000
Radii:
 Radius for O  used is    2.8724 Bohr (=   1.5200 Ang.)
 Radius for H  used is    2.2677 Bohr (=   1.2000 Ang.)
Calculating surface                               ...        done! (  0.0s)
GEPOL surface points                              ...          243
GEPOL Volume                                      ...     131.2421
GEPOL Surface-area                                ...     129.8207
Calculating surface distance matrix               ...        done! (  0.0s)
Performing Cholesky decomposition & store         ...        done! (  0.0s)
Overall time for CPCM initialization              ...                 0.1s
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
  0    -76.2097629069   0.000000000000 0.08659684  0.00906807  0.2797209 0.7000
  1    -76.2486680628  -0.038905155881 0.04503813  0.00472605  0.1039758 0.7000
                               ***Turning on DIIS***
  2    -76.2601034942  -0.011435431409 0.03875242  0.00430666  0.0140013 0.0000
  3    -76.2825878889  -0.022484394617 0.02282512  0.00248390  0.0581972 0.0000
  4    -76.2859335937  -0.003345704853 0.00499529  0.00061958  0.0111817 0.0000
                      *** Initiating the SOSCF procedure ***
                           *** Shutting down DIIS ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  5    -76.28608510  -0.0001515081  0.000812  0.000812  0.001618  0.000185
               *** Restarting incremental Fock matrix formation ***
  6    -76.28609303  -0.0000079294  0.000363  0.001003  0.001479  0.000141
  7    -76.28609305  -0.0000000155  0.000558  0.000221  0.000520  0.000051
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   8 CYCLES          *
               *****************************************************


SMD CDS free energy correction energy :                 1.45251     Kcal/mol
Total Energy after SMD CDS correction =               -76.283779209 Eh

----------------
TOTAL SCF ENERGY
----------------

Total Energy       :          -76.28377921 Eh           -2075.78716 eV

Components:
Nuclear Repulsion  :            9.08429710 Eh             247.19629 eV
Electronic Energy  :          -85.36807631 Eh           -2322.98345 eV
One Electron Energy:         -122.97092774 Eh           -3346.20906 eV
Two Electron Energy:           37.60285144 Eh            1023.22561 eV
CPCM Dielectric    :           -0.01566093 Eh              -0.42616 eV
SMD CDS (Gcds)     :            0.00231472 Eh               0.06299 eV

Virial components:
Potential Energy   :         -152.03201344 Eh           -4137.00141 eV
Kinetic Energy     :           75.74823424 Eh            2061.21424 eV
Virial Ratio       :            2.00707007


DFT components:
N(Alpha)           :        5.000000083076 electrons
N(Beta)            :        5.000000083076 electrons
N(Total)           :       10.000000166152 electrons
E(X)               :       -8.931637559555 Eh       
E(C)               :       -0.331338511791 Eh       
E(XC)              :       -9.262976071346 Eh       
DFET-embed. en.    :        0.000000000000 Eh       
CPCM Solvation Model Properties:
Surface-charge          :           -0.01263955
Charge-correction       :           -0.00004877 Eh              -0.00133 eV
Free-energy (cav+disp)  :            0.00231472 Eh               0.06299 eV

---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -8.7791e-07  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    1.3243e-04  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    1.7679e-05  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    5.7087e-05  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    1.3437e-04  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (water_spe_sol.gbw) ****
             **** DENSITY water_spe_sol.scfp WAS UPDATED ****
             **** ENERGY FILE WAS UPDATED (water_spe_sol.en.tmp) ****
             **** THE GBW FILE WAS UPDATED (water_spe_sol.gbw) ****
             **** DENSITY water_spe_sol.scfp WAS UPDATED ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -18.735440      -509.8172 
   1   2.0000      -0.882906       -24.0251 
   2   2.0000      -0.449007       -12.2181 
   3   2.0000      -0.312087        -8.4923 
   4   2.0000      -0.233725        -6.3600 
   5   0.0000       0.050155         1.3648 
   6   0.0000       0.129120         3.5135 
   7   0.0000       0.525200        14.2914 
   8   0.0000       0.586666        15.9640 
   9   0.0000       0.852671        23.2024 
  10   0.0000       0.861474        23.4419 
  11   0.0000       0.956090        26.0165 
  12   0.0000       1.050513        28.5859 
  13   0.0000       1.303431        35.4682 
  14   0.0000       1.358931        36.9784 
  15   0.0000       1.497923        40.7606 
  16   0.0000       1.736591        47.2550 
  17   0.0000       2.140559        58.2476 
  18   0.0000       2.184024        59.4303 
  19   0.0000       2.864857        77.9567 
  20   0.0000       2.907621        79.1204 
  21   0.0000       3.089366        84.0659 
  22   0.0000       3.398782        92.4856 
  23   0.0000       3.721026       101.2543 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 O :   -0.374245
   1 H :    0.187155
   2 H :    0.187090
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 O s       :     3.682642  s :     3.682642
      pz      :     1.697255  p :     4.682836
      px      :     1.301496
      py      :     1.684085
      dz2     :     0.001486  d :     0.008766
      dxz     :     0.000591
      dyz     :     0.002750
      dx2y2   :     0.003335
      dxy     :     0.000604
  1 H s       :     0.744613  s :     0.744613
      pz      :     0.016304  p :     0.068233
      px      :     0.035696
      py      :     0.016233
  2 H s       :     0.744665  s :     0.744665
      pz      :     0.025363  p :     0.068245
      px      :     0.017143
      py      :     0.025739


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 O :   -0.192591
   1 H :    0.096316
   2 H :    0.096275

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 O s       :     3.432498  s :     3.432498
      pz      :     1.693114  p :     4.743073
      px      :     1.366844
      py      :     1.683115
      dz2     :     0.002482  d :     0.017020
      dxz     :     0.000766
      dyz     :     0.005548
      dx2y2   :     0.007422
      dxy     :     0.000800
  1 H s       :     0.728680  s :     0.728680
      pz      :     0.042874  p :     0.175004
      px      :     0.089718
      py      :     0.042412
  2 H s       :     0.728718  s :     0.728718
      pz      :     0.070423  p :     0.175008
      px      :     0.033261
      py      :     0.071324


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 O      8.3742     8.0000    -0.3742     1.9850     1.9850     0.0000
  1 H      0.8128     1.0000     0.1872     0.9976     0.9976     0.0000
  2 H      0.8129     1.0000     0.1871     0.9977     0.9977     0.0000

  Mayer bond orders larger than 0.100000
B(  0-O ,  1-H ) :   0.9925 B(  0-O ,  2-H ) :   0.9925 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 2 sec 

Total time                  ....       2.376 sec
Sum of individual times     ....       2.038 sec  ( 85.8%)

Fock matrix formation       ....       1.413 sec  ( 59.5%)
  Split-RI-J                ....       0.285 sec  ( 20.2% of F)
  XC integration            ....       0.307 sec  ( 21.7% of F)
    Basis function eval.    ....       0.013 sec  (  4.2% of XC)
    Density eval.           ....       0.013 sec  (  4.2% of XC)
    XC-Functional eval.     ....       0.015 sec  (  4.7% of XC)
    XC-Potential eval.      ....       0.005 sec  (  1.7% of XC)
Diagonalization             ....       0.140 sec  (  5.9%)
Density matrix formation    ....       0.004 sec  (  0.2%)
Population analysis         ....       0.039 sec  (  1.6%)
Initial guess               ....       0.055 sec  (  2.3%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.257 sec  ( 10.8%)
SOSCF solution              ....       0.074 sec  (  3.1%)
Grid generation             ....       0.056 sec  (  2.4%)

Maximum memory used throughout the entire SCF-calculation: 6.2 MB

-------------------------   --------------------
FINAL SINGLE POINT ENERGY       -76.283779209329
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... water_spe_sol.gbw
Electron density                                ... water_spe_sol.scfp
The origin for moment calculation is the CENTER OF MASS  = (-5.993827, -1.179295  0.045610)

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:     -0.04525       0.04592      -0.04483
Nuclear contribution   :      0.57352      -0.58041       0.56656
                        -----------------------------------------
Total Dipole Moment    :      0.52827      -0.53449       0.52173
                        -----------------------------------------
Magnitude (a.u.)       :      0.91485
Magnitude (Debye)      :      2.32536



--------------------
Rotational spectrum 
--------------------
 
Rotational constants in cm-1:    26.442708    14.345901     9.300255 
Rotational constants in MHz : 792732.440810 430079.286725 278814.632727 

 Dipole components along the rotational axes: 
x,y,z [a.u.] :    -0.000093     0.914849    -0.000006 
x,y,z [Debye]:    -0.000235     2.325362    -0.000016 

 

Timings for individual modules:

Sum of individual times         ...        3.019 sec (=   0.050 min)
GTO integral calculation        ...        0.376 sec (=   0.006 min)  12.5 %
SCF iterations                  ...        2.643 sec (=   0.044 min)  87.5 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 3 seconds 467 msec
